src/frontend/components/QuillEditor/QuillEditor.jsx

```diff
@@ -5,9 +5,11 @@ import "react-quill/dist/quill.snow.css";
 import "react-quill/dist/quill.core.css";
 import "./QuillEditor.css";
 const QuillEditor = ({ content, noteDataDispatch }) => {
+
   const handleChange = value => {
     noteDataDispatch({ type: "SET_CONTENT", payload: value });
   };
+
```

Comment:
One suggestion (not important one) - you could do a separate branch for formatting and refactor as changes such as adding/ removing line/ space becomes a misleading change in the PR for a feature.